### ISSUE TITLE
[ISSUE] Unnecessary Horizontal Scrollbar in Empty New Tiddler

### ISSUE THREAD

**Issue by kookma** [ahanniga/tiddly-dark#2]
![image](https://user-images.githubusercontent.com/830394/146057582-c55a4fe8-1929-4061-9651-c56782c074d7.png)

When creating a new tiddler, there a horizontal scrollbar at the bottom!
When you press enter in body (text field), the scrollbar disappears!

I assume the Theme Tweaks may needs some adjustment!

**Comment by ahanniga** [comment]
Hello, I ask which browser you are using? Thanks!

**Comment by kookma** [comment]
I tested in Chrome ad Edge 96.
I am using Node.JS 14 with TW 5.2.1 on Windows 10

**Comment by kookma** [comment]
I also checked with Firefox and Opera on Win10 + TW5.2.1 + Node.JS14

## Firefox 94
Shows the scrollbar but in dark
![image](https://user-images.githubusercontent.com/830394/146139732-f7db77a8-d18d-4a0f-a19e-265ff061c1c3.png)

## Opera 81
![image](https://user-images.githubusercontent.com/830394/146139986-f94c017e-4ac1-4f97-a944-5fc7cde2a13c.png)

**Comment by ahanniga** [comment]
Sorry this has been outstanding for a while. I was using Tiddly-dark on Linux and I saw similar issue, but with the sidebar. Adding this to the bottom of the Theme Tweaks tiddler will fix it up:

```css
/* Hopefully this fixes the ugly scrollbars on Linux and issue #2 */
::-webkit-scrollbar-track
{
    -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
    border-radius: 10px;
    background-color: #22272e;
}

::-webkit-scrollbar
{
    width: 10px;
    height: 10px;
    background-color: #22272e;
}

.tc-sidebar-scrollable::-webkit-scrollbar
{
    width: 0px;
    height: 0px;
}

::-webkit-scrollbar-thumb
{
    border-radius: 10px;
    -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,.3);
    background-color: #444c56;
}

::-webkit-scrollbar-corner {
    background-color: #22272e;
} {
   width: 200px;
   height: 400px;
   overflow: scroll;
   white-space: nowrap;
}
```
If no further scrollbar issues arise I'll merge it in.

**Comment by kookma** [comment]
Hi @ahanniga 
Testing on WIn10, Chrome 101, it seems the bug has been fixed.
So I think go for this fix.

I did not test on small screen.
